core/jazz_create-serverless-service/config/prod-config.json

```diff
@@ -7,5 +7,10 @@
     "SERVICE_API_RESOURCE": "/prod/jazz/services",
     "TOKEN_URL": "/prod/jazz/login",
     "SERVICE_USER": "{jazz_admin}",
-    "TOKEN_CREDS": "{jazz_admin_creds}"
+    "TOKEN_CREDS": "{jazz_admin_creds}",
+    "DEPLOYMENT_TARGETS": {
```

Comment:
I think this list should come from installer-vars.json. Creating an issue for this.

Reply:
sure. lets not block this PR for this as this was reviewed. Lets take this as an improvement

Reply:
created a story for this.